examples/wasm-benchmark/src/index.ts

```diff
@@ -55,16 +55,6 @@ render(state);
     state.licenseKey = licenseKey;
     render(state);
 
-    // We pre-fetch some assets in order to not affect the benchmark results.
-    const preFetchAssets = [
-       "./vendor/@nutrient-sdk/viewer/nutrient-viewer-lib/nutrient-viewer.wasm.js",
-       "./vendor/@nutrient-sdk/viewer/nutrient-viewer-lib/nutrient-viewer.wasm",
-    ]
-      .filter(Boolean)
-      .map((asset) => fetch(asset));
-
-    await Promise.all(preFetchAssets);
-
```

Comment:
Why is this being removed 🤔

Reply:
Because it never worked: those are not the actual file names. If we consider it valuable, we would have to figure out the actual file names for the WASM and WASM glue code files, which change on every release, and pass them on here somehow (env vars or something). I'm not sure it's worth it.

Reply:
Makes sense, I got confused because I thought it was related to some vulnerability issue.